src/store.js

```diff
@@ -70,9 +70,11 @@ export function genCode() {
         continue
       }
       store.code[file] = ejs
-        .render(currentFiles[file], store.config)
-        .replaceAll(/\s{4}\n/gi, '\n')
-        .replaceAll(/(\n{3,})/gi, '\n\n')
+        .render(
+          currentFiles[file].replace(/([\s\n]+#:::\s)/gi, '#::: '),
+          store.config
+        )
+        .replace(/\s{4}#$/gim, '')
```

Comment:
Could you please add a comment on what happends here and how do we trim lines and the assumption made.